### DISCUSSION TITLE
v13:  Importing a migration does not perform any database updates

### DISCUSSION THREAD

**Discussion by BarryFogarty** [Jumoo/uSyncMigrations#266] [Q&A]
Background:  I have performed a v8 - v13 Umbraco database upgrade, so my test site backoffice is already populated with data.  I'm now trying to use uSync Migrations to convert Nested Content properties to Blocklists (mainly) in-place.   NB I have disabled the content and media handlers so as to only work with settings.

I have created a default initial site migration using 'Convert Nested Content to BlockList and Grid to BlockGrid'.  The uSync files look correct when I inspect after running this conversion - they are indeed converting NC's to Blocklists.  However, when I report or import this migration, no DB schema changes are actually made.  There are no errors in the logs.

It should work the way I describe, right?  I could copy my migrated uSync files over the existing files in `~/uSync/v9/` but my reading of the docs is that should not be necessary.  I could use the migrated uSync files on a fresh site, but again, I am trying to keep the process as simple as possible as it will be replayed in the future with a fresh copy of the live site data.

UPDATE:  I have also tried using the 'Convert Nested Content to BlockLists' migration plan, as well as with content and media handlers enabled - but in both cases the results were the same.

**Comment by ProNotion** [comment]
@BarryFogarty I've not tried in v13 have done many in v10 on the way to v12. Is this part of your migration plan the first step? If not then I have found that I have needed to do a full uSync export after each migration step is imported before starting on the next. Not sure if that might help?

**Comment by maudTriple** [comment]
Did you find a solution for this? currently having the same problem, the types are updated but the db is not.
